src/main/web_src/src/components/ui/form/inputs/select/Select.js

```diff
@@ -31,15 +30,6 @@ export const Select = ({
 }) => {
 	let _value = options.filter(o => o.value === value)
 
-	/**
-	 * CUSTOM MULTI LOGIC
-	 * react-select-virtualized støtter foreløpig ikke multi-select
-	 * så denne biten må gjøres litt manuelt
-	 */
-	if (isMulti) {
```

Comment:
Kan denne egentlig fjernes? 🤔  For nå ser det ikke ut til at "multi"-felter fungerer lenger.